linkerd/meshtls/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![deny(rust_2018_idioms, clippy::disallowed_methods, clippy::disallowed_types)]
+#![allow(clippy::large_enum_variant)]
```

Comment:
is there a specific large enum variant in this crate, or are there multiple? would it be better to put the allow attribute on the offending enum, rather than crate-wide?

Reply:
This crate consists only of enums over the feature-flagged variants. It's complaining that the Unimplemented variants are 0B and the others are not.

Reply:
ah, got it!